### PR TITLE
Add support for transaction type 'LUOTON MAKSU'

### PR DESCRIPTION
**Pull request by ahuopana**

### CODE REVIEW

File: banking/statements/osuuspankki/__init__.py [hunk 1]
```diff
@@ -17,7 +17,8 @@
    "VUOKRA": "PAYMENT",
    "VERONPALAUTUS": "DIRECTDEP",
    "PALKKA": "DIRECTDEP",
-   "ARVOPAPERI": "OTHER"
+   "ARVOPAPERI": "OTHER",
+   "LUOTON MAKSU": "REPEATPMT"
```
Comment (petri) [inline banking/statements/osuuspankki/__init__.py:21]: Are you certain that mapping of "LUOTON MAKSU" to "REPEATPMT" is always correct, or the most common case? Do you have any documentation suggesting that?

**Comment by ahuopana** [comment]
No supporting documentation available.

In (personal finance) loan agreements, account holder typically authorizes
bank to make the payment on behalf of account holder on certain date until
loan is fully paid. With mortgages might be up to ~30 years. Thus repeat
payment was selected.

No objections if you prefer some other transaction type.

Regards,
Antti




On 1 Mar 2018 19:27, "Petri Savolainen" <notifications@github.com> wrote:

*@petri* commented on this pull request.
------------------------------

In banking/statements/osuuspankki/__init__.py
<https://github.com/koodaamo/banking.statements.osuuspankki/pull/5#discussion_r171609689>
:

> @@ -17,7 +17,8 @@
    "VUOKRA": "PAYMENT",
    "VERONPALAUTUS": "DIRECTDEP",
    "PALKKA": "DIRECTDEP",
-   "ARVOPAPERI": "OTHER"
+   "ARVOPAPERI": "OTHER",
+   "LUOTON MAKSU": "REPEATPMT"

Are you certain that mapping of "LUOTON MAKSU" to "REPEATPMT" is always
correct, or the most common case? Do you have any documentation suggesting
that?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/koodaamo/banking.statements.osuuspankki/pull/5#pullrequestreview-100492667>,
or mute the thread

**Comment by petri** [comment]
Ok I will merge this. We can always change the mapping later if need be.

**Comment by petri** [comment]
Implemented in 21aa35b

**Pull request closed without merging**